FrostyModSupport/FrostyModExecutor.cs

```diff
@@ -272,7 +272,8 @@ private void ProcessModResources(IResourceContainer fmod)
                     resource.FillAssetEntry(bEntry);
 
                     addedBundles.TryAdd(bEntry.SuperBundleId, new HashSet<string>());
-                    addedBundles[bEntry.SuperBundleId].Add(bEntry.Name);
+                    lock (addedBundles[bEntry.SuperBundleId])
```

Comment:
add braces and put the hashset in a variable please so the getter does not get called twice